woocommerce/woocommerce-admin-test-helper
# Add ability to view & edit option content

This PR adds the ability to view & edit option content.

![Screen Shot 2021-03-30 at 5 19 40 PM](https://user-images.githubusercontent.com/4723145/113072569-3e2c6080-917c-11eb-8d26-4aca30513156.jpg)

## Testing Instructions

1. Clone this repository
2. Run `npm install` && `npm start`
3. Navigate to Tools -> WCA Test Helper
4. You should see the new "Edit" button.
